demo/data/tree.json

```diff
@@ -12,12 +12,18 @@
           {
             "key": "gc-1",
             "icon": "ff ff-diamond",
-            "text": "Grandchild 1"
+            "text": "Grandchild 1",
+            "dataAttr": {
+              "fqname": "Some selected value of node 1"
```

Comment:
I don't think you need to introduce a miq-specific attribute here if you can use the already existing ones.